### PR TITLE
[fix-backend-gettext] Use more string literals for user-facing errors

### PR DESCRIPTION
**Pull request by wlonk**
This is in aid of getting makemessages to gather them correctly. It cannot see anything but single string literals inside gettext calls.

I would love another set of eyes to make sure I'm not missing other backend-generated user-facing errors we should be able to translate.

cats

### CODE REVIEW

File: metashare/api/sf_run_flow.py [hunk 1]
```diff
@@ -64,15 +64,29 @@ def refresh_access_token(*, config, org_name, scratch_org):
...
+                # literals should be parsed by the AST into a single
+                # string literal and picked up by makemessages, but
+                # that's a gripe for another day). We have relatively
+                # few errors that propagate directly from the backend
+                # like this, but when we do, this is the pattern we
+                # should use.
+                #
+                # This is also why we repeat the first sentence in three
+                # places.
+                f"Are you certain that the org still exists? If you need support, your job ID is {job_id}."  # noqa: E501
```
Comment (jgerigmeyer) [inline metashare/api/sf_run_flow.py]: I assume these count as "string literals" even with the `{job_id}` dynamic aspect?

File: metashare/api/sf_run_flow.py [hunk 2]
```diff
@@ -64,15 +64,26 @@ def refresh_access_token(*, config, org_name, scratch_org):
...
+            # pick it up correctly, we need it to be a single,
+            # unbroken, string literal (even though adjacent string
+            # literals should be parsed by the AST into a single
+            # string literal and picked up by makemessages, but
+            # that's a gripe for another day). We have relatively
+            # few errors that propagate directly from the backend
+            # like this, but when we do, this is the pattern we
+            # should use.
+            #
+            # This is also why we repeat the first sentence.
```
Comment (davisagli) [inline metashare/api/sf_run_flow.py]: The first sentence could be set up as a separate message, e.g. _("sentence 1") + _("sentence 2")

Comment (wlonk) [reply metashare/api/sf_run_flow.py]: Alas, you can't concatenate the results of from `django.utils.translation.gettext_lazy`.

File: metashare/api/sf_run_flow.py (hunk 1 above)
Comment (wlonk) [reply metashare/api/sf_run_flow.py]: Yep. That leaves placeholders for variables for the translator to include.

File: metashare/api/sf_run_flow.py (hunk 2 above)
Comment (davisagli) [reply metashare/api/sf_run_flow.py]: Well that's disappointing!

Comment (wlonk) [reply metashare/api/sf_run_flow.py]: I know, right? I wonder if the non-lazy version gets around that, though. Just occurred to me.

**Pull request merged**